Documentation/upgrade.md

```diff
@@ -60,10 +60,11 @@ Most of the health verification checks for your cluster during the upgrade proce
 For more information about how to run the toolbox, please visit the [Rook toolbox readme](./toolbox.md#running-the-toolbox-in-kubernetes).
 
 ### Pods all Running
-In a healthy Rook cluster, the operator and all Rook namespace pods should be in the `Running` state and have few, if any, pod restarts.
+In a healthy Rook cluster, the operator, the agents and all Rook namespace pods should be in the `Running` state and have few, if any, pod restarts.
 To verify this, run the following commands:
 ```bash
-kubectl get pods -l app=rook-operator
+kubectl -n rook-system get pods -l app=rook-operator
```

Comment:
instead of doing the rook-operator and rook-agent individually, this documentation can probably just do `kubectl -n rook-system get pod`.  That way, if any new pods are added to the rook-system namespace, the documentation doesn't need to get updated again.

Note we are doing that for the rook namespace already instead of individually getting each mon, osd, rgw, etc.

Reply:
done